.gitattributes

```diff
@@ -10,6 +10,3 @@ doc/cli.markdown text eol=lf
 # crlf for the eol conversion test files
 tests/test-data/projects/docker-compose/basic/service2/file2-crlf.sh eol=crlf
 tests/test-data/projects/no-docker-compose/basic/src/windows-crlf.sh eol=crlf
-
-# Prevent auto merging of the npm-shrinkwrap.json file: see notes in CONTRIBUTING.md
-/npm-shrinkwrap.json merge=binary
```

Comment:
[As discussed in flowdock](https://www.flowdock.com/app/rulemotion/d-build-pipelines/threads/UbtLhou_pLyZQENwZMbWXL5cywQ), we have learned that balena CI's rebase script is this [rebase.sh](https://github.com/product-os/scripts/blob/master/meta/rebase.sh), which uses plain `git fetch`, `git checkout` and `git rebase` to do the rebase, and takes the `.gitattributes` file into account so that the `npm-shrinkwrap.json` will not be merged automatically. Here's a sample output of the CI's rebase job, for a test pull request (#1984):

https://ci.balena-dev.com/teams/main/pipelines/github-events-balena-io/jobs/rebase-event/builds/264
```
Using index info to reconstruct a base tree...
M    npm-shrinkwrap.json
Falling back to patching base and 3-way merge...
warning: Cannot merge binary files: npm-shrinkwrap.json (HEAD vs. test-rebase)
Auto-merging npm-shrinkwrap.json
CONFLICT (content): Merge conflict in npm-shrinkwrap.json
error: Failed to merge in the changes.
Patch failed at 0001 test-rebase
hint: Use 'git am --show-current-patch' to see the failed patch
Resolve all conflicts manually, mark them as resolved with
"git add/rm <conflicted_files>", then run "git rebase --continue".
You can instead skip this commit: run "git rebase --skip".
To abort and get back to the state before "git rebase", run "git rebase --abort".
```
When the auto merge of `npm-shrinkwrap.json` fails as above (because of the `.gittattributes` file), the `(at)balena-ci rebase` command fails and the PR is not automatically rebased.